eng/common/TestResources/New-TestResources.ps1

```diff
@@ -528,7 +528,9 @@ try {
         $templateParameters.Add('azsdkPipelineSubnetList', @($env:PoolSubnet))
     }
     # Some arm/bicep templates may want to change deployment settings (e.g. local auth) in sandboxed TME tenants
-    $templateParameters.Add('supportsSafeSecretStandard', ($context.Tenant.Name -notlike '*TME*'))
+    # The pipeline account context does not have the .Tenant.Name property, so check against subscription via
+    # naming convention instead.
+    $templateParameters.Add('supportsSafeSecretStandard', ($context.Subscription.Name -notlike '*TME*'))
```

Comment:
This seems like it might have false positive matches if tme shows up anywhere in the sub name. While it isn't ideal perhaps just match on the TME tenant id.

Reply:
@weshaggard I pushed a new approach.